src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables.scss";
+
+@import "./blocks/reset.scss";
```

Comment:
Better put reset before other imports (as I remember)

Reply:
I thought blocks must be after utils...